### PR TITLE
Allows Locksmith to utilize env variable for DB config

### PR DESCRIPTION
**Pull request by akeem**
As part of the follow up items after landing locksmith in the code base, this pull request allows Locksmith to utilize environment variable to specify database details. 

This represent the first portion of the overall configuration change and set the direction of our staging and subsequent production deployments

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [x] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

PS: [Read how to write the perfect pull request](https://blog.github.com/2015-01-21-how-to-write-the-perfect-pull-request/)

### CODE REVIEW

File: locksmith/config/config.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_NAME,
+    host: process.env.DB_HOSTNAME,
+    logging: false, 
+    dialect: 'postgres',      
+  },
+  production: {
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
```
Comment (cellog) [inline locksmith/config/config.js]: in order to prepare for using docker or kubernetes secrets, which are always represented as a file in a ramfs, I'd add some further logic here.

top of the file:

```js
const fs = require('fs')
```

then:

```js
    password: process.env.DB_PASSWORD_FILE ? fs.readFileSync(process.env.DB_PASSWORD_FILE) : process.env.DB_PASSWORD
```

Then we will be golden

Comment (cellog) [reply locksmith/config/config.js]: unless, of course, you plan to inject the .env file into the locksmith root directly, which is possible, but a bit magic-y. Docker secrets live in /run/secrets by default

File: locksmith/package.json [hunk 2]
```diff
@@ -22,6 +22,7 @@
     "babel-eslint": "^10.0.1",
     "body-parser": "^1.18.3",
     "cors": "^2.8.5",
+    "dotenv": "^6.2.0",
```
Comment (cellog) [inline locksmith/package.json]: probably do these config changes in a separate pre-PR

**Comment by akeem** [comment]
@cellog thanks for the notes, but i'm actually still working on this

**Pull request merged**